int8 id
bool state
time timestamp
